pyaedt/modules/monitor_icepak.py

```diff
@@ -527,8 +526,8 @@ def insert_monitor_object_from_dict(self, monitor_dict, mode=0):
         mode : int
             Integer to select the information to handle. To identify faces, vertices, surfaces and object to which the
             monitor is to be assigned it is possible to use:
-                ids and names, mode=0, required dict keys: "Name", "Type", "ID", "Quantity".
-                positions, mode=1, required dict keys: "Name", "Type", "Geometry Assignment", "Location", "Quantity".
+            - ids and names, mode=0, required dict keys: "Name", "Type", "ID", "Quantity".
+            - positions, mode=1, required dict keys: "Name", "Type", "Geometry Assignment", "Location", "Quantity".
```

Comment:
I'm not sure if this is a list of options that should be in literal strings or general descriptions. In text, we are using ID and IDs rather than id and ids.